resolver_conn_wrapper_test.go

```diff
@@ -122,21 +125,18 @@ func (s) TestDialParseTargetUnknownScheme(t *testing.T) {
 }
 
 func testResolverErrorPolling(t *testing.T, badUpdate func(*manual.Resolver), goodUpdate func(*manual.Resolver), dopts ...DialOption) {
-	defer func(o func(int) time.Duration) { resolverBackoff = o }(resolverBackoff)
-
-	boIter := make(chan int)
```

Comment:
Can we bring this back now?

Reply:
Done. I hope i've gotten back the right thing.